AbletonMCP_Remote_Script/handlers/tracks.py

```
@@ -0,0 +1,658 @@
+
+
+def create_midi_track_with_simpler(song, track_index, clip_index, ctrl=None):
+    """Create a new MIDI track with a Simpler containing an audio clip's sample."""
+    try:
+        _, clip = get_clip(song, track_index, clip_index)
+        if not clip.is_audio_clip:
+            raise ValueError("Clip is not an audio clip")
+        try:
+            from Live.Conversions import create_midi_track_with_simpler as _create
+        except ImportError:
+            raise Exception("create_midi_track_with_simpler requires Live 12+")
```

Ruff (0.15.0)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)